packages/plugin-ext/src/main/browser/selection-provider-command.ts

```diff
@@ -0,0 +1,55 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { Command, CommandContribution, CommandRegistry } from '@theia/core/lib/common/command';
+import { inject, injectable } from 'inversify';
+import { UriAwareCommandHandler, UriCommandHandler } from '@theia/core/lib/common/uri-command-handler';
+import URI from '@theia/core/lib/common/uri';
+import { SelectionService } from '@theia/core';
+import { WorkspaceService } from '@theia/workspace/lib/browser';
+
+export namespace SelectionProviderCommands {
+    export const GET_SELECTED_CONTEXT: Command = {
+        id: 'workspace:selectedContext',
```

Comment:
It would be nice to prefix Theia specific plugin command that they don't collide eventually with Theia core commands, i.e. `theia.plugin.workspace.selectedContext`. And make it clear that they are confirm to theia plugin API, for instance using vscode uris.

cc @benoitf